src/python/grpcio_tests/tests/protoc_plugin/_python_plugin_test.py

```diff
@@ -503,5 +504,118 @@ def half_duplex_request_iterator():
         service.server.stop(None)
 
 
+@unittest.skipIf(sys.version_info[0] < 3, "Unsupported on Python 2.")
+class SimpleStubsPluginTest(unittest.TestCase):
+    servicer_methods = _ServicerMethods()
```

Comment:
nit: Is `setUpClass` an option?

Reply:
Yes. But it's not the pattern that the rest of the tests in this file are using. I decided to prioritize meshing well with the rest of the file over going with something slightly more conventional.